cryptokit/block.py

```diff
@@ -124,7 +126,7 @@ def from_gbt(cls, retval, coinbase, extra_length=0, transactions=None):
         if transactions is None:
             transactions = []
         coinbase1, coinbase2 = coinbase.assemble(split=True)
-        inst = cls()
+        inst = cls(pos)
```

Comment:
Just as a heads up this is probably not what you wanted. The first argument on `__init__` is `raw`, so this is passing the `raw` argument the value of `pos`.

Should actually be `inst = cls(pos=pos)`